docs/source/about/advanced_usage.rst

```diff
@@ -883,6 +883,22 @@ keybindings
 
       ping_timeout = 10000
 
+`outgoing_bandwidth <https://localhost:47990/config/#outgoing_bandwidth>`__
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+   Bandwidth limitation in bytes/second to manually deal with local network throttling when the host has a greater data rate than the stream.
+   Note: for unlimited bandwidth use 0
+   Example: 1Gbps = 125000000 MBps
```

Comment:
```suggestion
**Description**
   Bandwidth limitation in MB/s. Throttling the outgoing traffic can be necessary if there is a speed mismatch anywhere
   in the network path between the host and the client. E.g. the host is connected via 2.5Gbps LAN, but the client uses
   a 200 Mbps WLAN connection. The bandwidth limit should ideally be set in accordance to the slowest segment on the
   network path. It should also be at least two to three times the requested stream bandwidth.
   Note: for unlimited bandwidth use 0
   Example: 1Gbps = 125 MB/s
```

I agree with @gschintgen about using MB/s instead of Bytes per second, as well as improving the description detailing when this setting would be helpful.

All other changes look okay to me.

Reply:
Just in order to avoid further confusion: Moonlight has Mbps (megabits per second) as is often the case for streaming/networking.

(b for bits, B for bytes. This convention is not universally followed but it's quite sensible.)

I'd rather have those instead of MB/s or MiB/s. (megabytes and mebibytes respectively).

Reply:
I agree with that. Let's use what moonlight uses.